Gemfile

```diff
@@ -48,7 +48,7 @@ gem 'rake', '~> 12'
 
 gem 'mime-types', "~> 3"
 
-gem 'capybara', '~> 2.13', require: false
+gem 'capybara', '>= 2.13', '< 4.0', require: false
```

Comment:
I'd still prefer `'~> 2.13', '~> 3.0'` its semantically the same but a bit clearer in my opinion. Unless its not valid for some reason?

Reply:
Bundler seems to combine the options passed with "and", so this fails with:
```
Could not find gem 'capybara (~> 2.13, ~> 3.0)' in any of the gem sources listed in your Gemfile.
```

Reply:
Fair enough!